velox/exec/fuzzer/WindowFuzzer.h

```diff
@@ -124,6 +160,27 @@ class WindowFuzzer : public AggregationFuzzerBase {
 
     void print(size_t numIterations) const;
   } stats_;
+
+  // For k PRECEDING/FOLLOWING frame bounds in RANGE mode, where k is constant,
+  // Velox uses a column with the pre-computed offset values as the frame bound,
+  // instead of the constant k value which is used by Presto. This vector
+  // represents the constant value of the frame bound for such frames, with each
+  // pair in the vector corresponding to a window in the plan node. A pair is
+  // used to represent the frame start and end bounds since they both could be
+  // of type k-RANGE. When Presto is used as reference DB for verification, the
+  // frame bound values are obtained from this variable. Eg: If the window plan
+  // node operator being tested by the window fuzzer has two window operators
+  // i.e window({"sum(c) OVER (ORDER BY s0 FRAME BETWEEN k2 PRECEDING AND k3
+  // FOLLOWING)", "avg(c) OVER (ORDER BY s1 DESC RANGE BETWEEN UNBOUNDED
+  // PRECEDING AND k9 FOLLOWING)"}); where the offset columns k2, k3, and k9 are
+  // pre-computed as k2[row] = s0[row] - 2, k3[row] = s0[row] + 3, k9[row] =
+  // s1[row]- 9, the variable prestoFrames_ corresponding to this window plan
+  // node would look be: std::vector{ std::pair{"2", "3"}, std::pair{"", "3"} },
+  // and the frame clause in Presto, corresponding to the two window operators
+  // would be "RANGE BETWEEN 2 PRECEDING AND 3 FOLLOWING", "RANGE BETWEEN
+  // UNBOUNDED PRECEDING AND 9 FOLLOWING". This frame clause can be constructed
```

Comment:
Why not use string UNBOUNDED PRECEDING in the frame instead of "". Would be cleaner.